CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## v1.4.0
 * Added `logout` command
 * Fixed concurrent read/write operations when accessing context.storage objects
+* Add option to ignore the extension-config.json file of an extension so the config.json won't get updated on start (backend) or on change of extension-config.json; use env `IGNORE_EXT_CONFIG_FOR` with a comma separeted list like this: `IGNORE_EXT_CONFIG_FOR="extId1(,extId2,extId3,...)"`
```

Comment:
I guess we should start beginning all of these entries with past tense verbs: Added, Fixed, Changed, ...

Typo in "separeted", must be "separated".

Reply:
done